WordPress/gutenberg
# [RNMobile] Gallery - Only run the linkTo saving on the web since it's not mobile related

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2735
## Description
https://github.com/WordPress/gutenberg/pull/25582 introduced changes to utilize the `image_default_link_type` from `options.php` to set the default link for galleries. 

The issue is that the logic being utilized in the `useEffect` hook that triggers the saving of the link doesn't work with mobile 
https://github.com/WordPress/gutenberg/blob/6f82c4bb16a5afb5d796a8f51905ca27276fe283/packages/block-library/src/gallery/edit.js#L304

So the current solution is to disable this on mobile and only run it on the web. 

## How has this been tested?

This has been tested using the instructions below. 

1. Add a Gallery Block to the canvas. 
2. Press the undo button. 
3. Notice that the Gallery block is now removed. 

Previously, the block wouldn't be removed.

## Screenshots 

Before | After 
--------|-------
<kbd><img src="https://user-images.githubusercontent.com/1509205/96312484-56932f00-0fd1-11eb-8d35-3bc890e6ebea.gif" width="320"></kbd>   |    <kbd><img src="https://user-images.githubusercontent.com/1509205/96312500-5c891000-0fd1-11eb-96f2-406c71268cd6.gif" width="320"></kbd>
   
## Types of changes





## Checklist:
